src/interpreter.rs

```diff
@@ -176,21 +186,19 @@ impl<T: Clone> Stack<T> {
     }
 
     fn last_mut(&mut self) -> Result<&mut T, ScriptError> {
-        self.0.last_mut().ok_or(ScriptError::InvalidStackOperation)
+        let len = self.0.len();
+        self.0
+            .last_mut()
+            .ok_or(ScriptError::InvalidStackOperation(Some((0, len))))
     }
 
     fn last(&self) -> Result<&T, ScriptError> {
-        self.0.last().ok_or(ScriptError::InvalidStackOperation)
-    }
-
-    fn split_last(&self) -> Result<(&T, Stack<T>), ScriptError> {
         self.0
-            .split_last()
-            .ok_or(ScriptError::InvalidStackOperation)
-            .map(|(last, rem)| (last, Stack(rem.to_vec())))
+            .last()
+            .ok_or(ScriptError::InvalidStackOperation(Some((0, self.0.len()))))
     }
 
-    fn rremove(&mut self, start: usize) -> Result<T, ScriptError> {
+    pub fn rremove(&mut self, start: usize) -> Result<T, ScriptError> {
```

Comment:
Does this need to be public? If so, document it as appropriate in the changelog, and document the method itself.